### PR TITLE
[debugger] Bump protocol for this commit https://github.com/mono/mono/pull/19248

### PR DESCRIPTION
**Pull request by thaystg**
Bump protocol for this commit https://github.com/mono/mono/pull/19248 as suggested by @vargaz.

### CODE REVIEW

File: mono/mini/debugger-agent.c [hunk 1]
```diff
@@ -6961,6 +6961,11 @@ get_types_for_source_file (gpointer key, gpointer value, gpointer user_data)
 	}
 }
 
+static void add_error_string (Buffer *buf, const char *str) {
+	if (CHECK_PROTOCOL_VERSION (2, 56))
```
Comment (vargaz) [inline mono/mini/debugger-agent.c]: Coding convention.

**Comment by thaystg** [comment]
@monojenkins build failed

**Comment by thaystg** [comment]
@monojenkins build failed

**Comment by thaystg** [comment]
@monojenkins backport 2020-02

**Comment by thaystg** [comment]
@monojenkins backport 2020-02

**Pull request merged**